shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -92,5 +128,13 @@ object Time {
   }
 
   /** Current time */
-  def now(): Time = ???
+      def now() : Time = {
```

Comment:
That's not a problem. On most operating systems this function will behave correctly anyway. If an operating system doesn't provide any accurate timestamps, every class relying on them would be affected (including `java.util.Date`).